ddtrace/internal/datadog/profiling/stack_v2/src/stack_renderer.cpp

```diff
@@ -64,7 +66,7 @@ StackRenderer::render_thread_begin(PyThreadState* tstate,
 }
 
 void
-StackRenderer::render_task_begin(std::string_view name)
+StackRenderer::render_task_begin(std::string_view)
```

Comment:
It's really not obvious to me just from the PR description or any comments in the code why this wasn't right before or why it's right now. As far as I can piece it together, the name here is derived when Echion enumerates tasks during a sample call. Is it not computing the name correctly? Are we adding the same name multiple times? Could you please include some description of the bug and why this fix works? And do we need to fix anything in your Echion fork, given that it's passing a no-longer-useful parameter here?

Reply:
It is not computing the name correctly. Updated the PR description. I don't think we need to fix anything in the Echion fork yet.